src/wasm/baseline/riscv32/liftoff-assembler-riscv32.h

```diff
@@ -1133,7 +1133,7 @@ bool LiftoffAssembler::emit_i64_popcnt(LiftoffRegister dst,
 }
 
 void LiftoffAssembler::emit_i32_mul(Register dst, Register lhs, Register rhs) {
```

Comment:
Would it be correct to remove this function?

Reply:
we can't remove it